tensorflow/core/framework/op_gen_lib.cc

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 #include "tensorflow/core/framework/op_gen_lib.h"
 
 #include <vector>
+#include <algorithm>
```

Comment:
Please keep the C++ system header includes sorted. 

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Done